common/lib/xmodule/xmodule/js/src/video/07_video_volume_control.js

```diff
@@ -50,13 +50,14 @@ function() {
                         '</span>',
                     '</span>',
                 '</button>',
-                '<div class="volume-slider-container" aria-hidden="true">',
+                '<div class="volume-slider-container" aria-hidden="true" title="' + gettext('Adjust video volume') + '">', // jshint ignore: line
```

Comment:
Instead of ignoring this line, can you break it up like the div on the next line does?